util/ConfigFileProcess.cpp

```diff
@@ -0,0 +1,39 @@
+#include "ConfigFileProcess.h"
+
+//
+// combining config, persistent_config and commandline args
```

Comment:
Don't place documentation at the definition, only to the declaration unless the symbol is limited to a file.